joomla/joomla-cms
# Fixed#36117 -- Underlined links in Joomla Update Section

Pull Request for Issue #36117 .

### Summary of Changes
Added Underline in the link in Joomla Update Section.


### Testing Instructions
See the link underlined in the Joomla update section and set the colour to monochrome for atum and see the link (underlined).


### Actual result BEFORE applying this Pull Request
No underlined link.

### Expected result AFTER applying this Pull Request
Underlined link in Joomla Update Section.

### Documentation Changes Required
No
